src/rgw/rgw_lua.cc

```diff
@@ -196,10 +196,11 @@ int list_packages(const DoutPrefixProvider *dpp, rgw::sal::Store* store, optiona
 
 int install_packages(const DoutPrefixProvider *dpp, rgw::sal::Store* store, optional_yield y, packages_t& failed_packages, std::string& output) {
   // luarocks directory cleanup
-  boost::system::error_code ec;
+  std::error_code ec;
   const auto& luarocks_path = store->get_luarocks_path();
-  boost::filesystem::remove_all(luarocks_path, ec);
-  if (ec.value() != 0 && ec.value() != ENOENT) {
+  std::filesystem::remove_all(luarocks_path, ec);
```

Comment:
why do you call `remove_all()` twice?

Reply:
@yuvalif sorry, forgot to remove the first one when adding the second one. fixed and repushed. could you take another look?